### ISSUE TITLE
util: 413 with very long input

### ISSUE THREAD

**Issue by srl295** [unicode-org/unicodetools#26]
I was trying to paste the CLDR `zh` exemplar set into the UnicodeSet tool, and got a 413.  4k of text expands to >8k of URI content.

The UnicodeSet submit button should detect that the input is above some threshhold, and submit the data as a POST.  You won't be able to bookmark the link (as with other use of the tool), but at least the intended operation will actually work.

**Comment by macchiati** [comment]
Yes, it definitely should be reworked; I've run into that limit before also.
I haven't ever used POST, so would need some pointers.

Mark

**Comment by roozbehp** [comment]
Pull request to fix at https://github.com/unicode-org/unicodetools/pull/1549
